windows/start.sh

```diff
@@ -84,6 +84,8 @@ docker () {
 }
 export -f docker
 
+set +e
```

Comment:
Honestly, I think it makes sense in this case since we're about to start an interactive shell, but I'd even move it down inside the first block of the `if` below (so it only applies during "interactive"), and +1 on @nathanleclaire's concern that it probably could use a comment.